### DISCUSSION TITLE
役期恢復一年 軍種梯次延續原編號

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#1002] [General]
這是一個會讓人很有想法的文章，為什麼要讓役期恢復成一年，會不會是代表台灣準備要備軍來抵抗中國了，還是單純只是覺得現在的太短了所以要延長，我個人是覺得是第一個，那這樣不就是在變想挑釁中國嗎?不知道政府這樣做是有什麼含意，反正我們還是16周拉，就看看接下來的事情會怎麼發展吧!
有興趣的可以去看看以下這篇文章
https://www.msn.com/zh-tw/news/opinion/%E5%BD%B9%E6%9C%9F%E6%81%A2%E5%BE%A9%E4%B8%80%E5%B9%B4-%E8%BB%8D%E7%A8%AE%E6%A2%AF%E6%AC%A1%E5%BB%B6%E7%BA%8C%E5%8E%9F%E7%B7%A8%E8%99%9F/ar-AA15ZyoM?ocid=msedgntphdr&cvid=cce40d3215864095af05fe818590d473

**Comment by swerfg03joe** [comment]
我是不知道政府到底在想什麼，但是真的不要亂改非常擾民，為了自己的政治利益整天挑釁對面然後看起來情況不妙了又要拖全民下水，但是沒改到我的部分，所以我暫時不會有太大的不滿

**Comment by WEI-CHEN-CHUN** [comment]
兵役從過去兩年變成四個月，如今情勢不同改成一年，或許是大多數人不習慣?

**Comment by s96015welson** [comment]
雖然男生當兵是義務，且也能理解需要延長到一年的理由，但真的幸好我自己是四個月兵，不然一年真的太久了，而且同時研發替代役也沒了，所以幾乎沒有其他選擇只能乖乖去當兵。

**Comment by harrylee1971** [comment]
好可憐的高三生們，要考學測還被送了一個一年大禮包，是我的話才不要進去關一年呢
